audiotrack.go

```diff
@@ -246,8 +256,12 @@ func (t *DecodingRemoteAudioTrack) SampleRate() int {
 }
 
 func (t *DecodingRemoteAudioTrack) Close() {
+	if t.pcmMWriter.String() != t.resampledPCMWriter.String() {
```

Comment:
Are you sure this is needed?

Reply:
Tried to trace the code, opusWriter closes the one it is initialized with. But, I couldn't see the same with resampler, so that might need explicit closing.